python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -155,8 +155,7 @@ def __init__(
         )
 
         self.mtcamera = mtcamera
-        self.ls_select_led_location = 9.96  # mm
-        self.ls_select_laser_location = 79.96  # mm
+        self.ls_select_locations = {"led": 9.96, "laser": 79.96}
```

Comment:
Since you are renaming this attribute, consider expanding `ls` -> `liner_stage`.

Also, what does "select_locations" means?  Consider more expressive names that clearly indicate what this attribute is.